shared/fs/row/common.js

```diff
@@ -39,6 +39,9 @@ const rowText = Styles.platformStyles({
     textOverflow: 'ellipsis',
     whiteSpace: 'nowrap',
   },
+  isMobile: {
+    maxWidth: '75%',
```

Comment:
I'm not sure this will scale well for different screen sizes, since the `EMPTY` tag is a fixed width.

Reply:
The best way to scale out is using percentage instead of pixels, the percentage will occupy the specified percentage of the screen unlike pixels which vary according to the device screen.

Reply:
In the docs, they advice you use points or percentages https://facebook.github.io/react-native/docs/layout-props#maxwidth

Reply:
<img width="436" alt="screen shot 2018-09-14 at 20 41 22" src="https://user-images.githubusercontent.com/10031957/45566190-9a4d6480-b85e-11e8-91a4-9f29e844b22f.png">

I've tried on the various screen sizes, it's working as expected